README.md

```diff
@@ -4,16 +4,16 @@ This action searches for team pings in issues and pull requests in an organizati
 
 # Input parameters
 
-- `token`: **Require:** An access token.
-- `team`: **Require:** Team name without the organization. For example: `docs-content-ecosystem`
-- `org`: **Require:** The organization to search for issues.'
-- `since`: The start date to search for team pings. Forma: {4 digit year}-{month}-{day}. For example: 2020-5-20'. Default: '2020-1-1'
-- `project-board`: 'The URL of the project board to place issues and pull requests. Must be an org project board.'
+- `token`: **Required:** An access token.
+- `team`: **Required:** Name of the team whose @mentions you wannt to search for. Use the team name without the organization. For example: `docs-content-ecosystem`.
+- `org`: **Required:** The organization to search for issues.
+- `since`: The start date to search for team pings. Form: {4 digit year}-{month}-{day}. For example: 2020-5-20. Default: '2020-1-1'
```

Comment:
```suggestion
- `since`: The start date to search for team pings. Form: {4 digit year}-{month}-{day}. For example: '2020-5-20'. Default: '2020-1-1'
```